### ISSUE TITLE
consider adding ordered list of channel dataset names to attributes

### ISSUE THREAD

**Issue by trautmane** [clbarnes/jeiss-convert#10]
I think it would be useful to add the ordered channel names derived [here](https://github.com/clbarnes/jeiss-convert/blob/main/jeiss_convert/utils.py#L352-L357) to a separate attribute so that you can always identify channel order with certainty.  The current convention of `AI<channel number>` with a limit of 5 channels works, but I think adding an explicitly ordered list is more future-proof.

Just a thought ...

**Comment by clbarnes** [comment]
Agreed, this would make for a good extra attribute in the same vein as the ISO dates and the named enum variants. So long as hdf5 is fine storing an array of strings of variable length, which I think it is. As this is a derived attribute rather than a normative one we could also comfortably strip or pad the original name with null bytes. Something like `ChannelNames`?

**Comment by clbarnes** [comment]
Fixed in https://github.com/clbarnes/jeiss-convert/commit/6f912da2f62bfa3d3c364052d381cd8bc1fc05af
